go/pvl/interp.go

```diff
@@ -195,11 +195,12 @@ func checkProofInner(m metaContext, pvlS string, service keybase1.ProofType, inf
 		}
 	}
 
-	if len(errs) == 0 {
+	switch len(errs) {
+	case 0:
 		return nil
-	} else if len(errs) == 1 {
+	case 1:
 		return errs[0]
-	} else {
```

Comment:
Switch is clearer. Fine to merge anyway. Hope disabling this rule works out

Reply:
Yeah, the new code is the switch, which I agree is clearer.  Did you mean you think the if-else chain is clearer?  I was actually glad this one was caught by the linter.

Reply:
oh, my bad. I thought the switch was old even though generally I can read diffs :D
I meant the switch is clearer. Maybe my understanding of the linter rule is backwards, it's been adding switches the whole time?

Reply:
Yep, adding switches.  Some people don't like that partly because it negates the built-in scoping of some if-else statements like:

```go
if err != nil {
	return err
} else if otherErr, ok := err.(OtherErr); ok {
  // do something with otherErr
} else {
  // do something else
}
```